### PR TITLE
[Android/iOS]: Add support for mapView.getZoom()

### LINKED ISSUE #938: Get the current map zoom level.
I have a use case where I wanted to press a cluster and zoom the map by `(currentZoom + 2)` but I realised that it's not possible at the moment to retrieve the current `zoomLevel`.

mapbox-gl-js has this method for example:
https://www.mapbox.com/mapbox-gl-js/api/#map#getzoom

It would be great to have it, probably on the `ref` `this.map.getZoom()`

### PR DESCRIPTION
**Pull request by itsjgf**
getZoom

Closes #938.

### CODE REVIEW

File: android/rctmgl/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView.java [hunk 1]
```diff
@@ -814,6 +814,20 @@ public void queryRenderedFeaturesAtPoint(String callbackID, PointF point, Filter
         mManager.handleEvent(event);
     }
 
+    public void getZoom(String callbackID) {
```
Comment (nitaliano) [inline android/rctmgl/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView.java:817]: mind removing the extra whitespaces in this method?

File: example/src/components/GetZoom.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  };
+
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      zoom: 9,
+    };
+
+
```
Comment (nitaliano) [inline example/src/components/GetZoom.js]: Same here we can get rid of some of the white spaces

File: example/src/components/GetZoom.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  }
+
+  onRegionDidChange = async () => {
```
Comment (nitaliano) [inline example/src/components/GetZoom.js]: We have normally been writing it like

```
async onRegionDidChange () => {}
``` 

mind updating so it matches the other places we use async/await

File: javascript/components/MapView.js [hunk 4]
```diff
@@ -508,6 +508,21 @@ class MapView extends React.Component {
...
+   *
+   * @example
+   * const zoom = await this._map.getZoom();
+   *
+   * @return {Number}
+   */
+
+  async getZoom (){
+    const res = await this._runNativeCommand('getZoom');
+
```
Comment (nitaliano) [inline javascript/components/MapView.js]: Same here could you remove the white space

**Review by nitaliano — CHANGES_REQUESTED** [review]
Other than a few nitpicks with code style. This PR looks great!

File: example/src/components/GetZoom.js (hunk 2 above)
Comment (itsjgf) [reply example/src/components/GetZoom.js]: done 😛

**Comment by nitaliano** [comment]
Thanks for updating will test this out then merge in

**Review by nitaliano — APPROVED** [review]

**Pull request merged**